src/app/components/dropdown/index.html

```diff
@@ -1,15 +1,176 @@
+        <h3>Behavior</h3>
+        <div class="sky-text-block sky-margin-stacked-content-section">
+            <p>The behavior of the dropdown menu is defined by the <a routerLink="/components/popover">popover</a> component.</p>
+        </div>
+        <h3>States</h3>
+        <div class="sky-text-block sky-margin-stacked-content-section">
+            <p>Dropdowns inherit the styling for their different states (hover, disabled, etc.) from <a routerLink="/components/button">SKY UX buttons</a>.</p>
+        </div>
+    </sky-demo-page-content>
+    
+    <sky-demo-page-content sectionHeading="Content">
+      <h3>Dropdown menu action text</h3>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>If an action clearly applies to the object associated with the context menu, use the following format for the action wording:</p>
+          <p><span class="sky-emphasized">&lt;Verb&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/context-menu.png" alt="context menu actions: View, Edit, Delete" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>For a dropdown button with one type of action, use the following format:</p>
+          <p>Button text: <span class="sky-emphasized">&lt;Verb&gt;</span></p>
+          <p>Menu item: <span class="sky-emphasized">&lt;Direct object&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/dropdown-single-action.png" alt="dropdown menu Edit: Addresses, Email addresses, Phone numbers" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>For a dropdown button with more than one type of action, use the following format:</p>
+          <p>Button text: <span class="sky-emphasized">More</span></p>
+          <p>Button icon: <span class="sky-emphasized">fa-ellipsis-h</span></p>
+          <p>Menu item: <span class="sky-emphasized">&lt;Verb&gt; &lt;direct object&gt;</span></p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img class="bordered" src="~/assets/img/guidelines/wording/more-button.png" alt="dropdown menu More: Place vendor on hold, Archive vendor" />
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      
+        <div class="sky-text-block sky-margin-stacked-content-section">
+        <p>Additional button text guidelines:</p>
+				<ul>
+					<li>Use sentence-style capitalization.</li>
+					<li>The direct object can be singular or plural depending on whether the action acts on one or multiple items.</li>
```

Comment:
I'll tweak this slightly to be a command like the other two items in the list. ... Also, "action acts" seems awkward, so I'll change "acts on" to "applies to" like we have in the first item in this section.